Clicker/Views/GroupControls/GroupControlsInfoCell.swift

```diff
@@ -1,100 +1,99 @@
     }
-
+    
     private func setupViews() {
         numMembersLabel = UILabel()
         numMembersLabel.textColor = UIColor.white.withAlphaComponent(0.75)
         numMembersLabel.font = ._16MediumFont
         numMembersLabel.textAlignment = .center
-        addSubview(numMembersLabel)
-
+        contentView.addSubview(numMembersLabel)
+        
         numPollsLabel = UILabel()
         numPollsLabel.textColor = UIColor.white.withAlphaComponent(0.75)
         numPollsLabel.font = ._16MediumFont
         numPollsLabel.textAlignment = .center
-        addSubview(numPollsLabel)
-
+        contentView.addSubview(numPollsLabel)
+        
         codeLabel = UILabel()
         codeLabel.textColor = UIColor.white.withAlphaComponent(0.75)
         codeLabel.font = ._16MediumFont
         codeLabel.textAlignment = .center
-        addSubview(codeLabel)
-
+        contentView.addSubview(codeLabel)
+        
         separatorLineView1 = UIView()
         separatorLineView1.backgroundColor = .clickerGrey14
-        addSubview(separatorLineView1)
-
+        contentView.addSubview(separatorLineView1)
+        
         separatorLineView2 = UIView()
         separatorLineView2.backgroundColor = .clickerGrey14
-        addSubview(separatorLineView2)
+        contentView.addSubview(separatorLineView2)
     }
-
+    
     private func setupConstraints() {
         numPollsLabel.snp.makeConstraints { make in
             make.center.height.equalToSuperview()
         }
-
+        
         separatorLineView1.snp.makeConstraints { make in
             make.trailing.equalTo(numPollsLabel.snp.leading).offset(-horizontalPadding)
             make.centerY.height.equalToSuperview()
             make.width.equalTo(separatorLineWidth)
         }
-
+        
         numMembersLabel.snp.makeConstraints { make in
             make.trailing.equalTo(separatorLineView1.snp.leading).offset(-horizontalPadding)
             make.height.centerY.equalToSuperview()
         }
-
+        
         separatorLineView2.snp.makeConstraints { make in
             make.leading.equalTo(numPollsLabel.snp.trailing).offset(horizontalPadding)
             make.centerY.height.equalToSuperview()
             make.width.equalTo(separatorLineWidth)
         }
-
+        
         codeLabel.snp.makeConstraints { make in
             make.leading.equalTo(separatorLineView2.snp.trailing).offset(horizontalPadding)
             make.height.centerY.equalToSuperview()
         }
     }
     
-    // MARK: - Configure
-    func configure(numMembers: Int, numPolls: Int, code: String) {
-        let memberDescriptor = numMembers == 1 ? "member" : "members"
-        let pollsDescriptor = numPolls == 1 ? "poll" : "polls"
-        numMembersLabel.text = "\(numMembers) \(memberDescriptor)"
-        numPollsLabel.text = "\(numPolls) \(pollsDescriptor)"
-        codeLabel.text = "Code: \(code)"
+    func configure(information: GroupControlsInfoModel) {
```

Comment:
```suggestion
    func configure(for information: GroupControlsInfoModel) {
```
i think it's swiftier to have configure functions that read more like "sentences"